### PR TITLE
skip not covered lines on node master

### PR DESCRIPTION
**Pull request by mcollina**
Alternative fix for the code coverage in master/node 10. See #91.

### CODE REVIEW

File: lib/request.js [hunk 1]
```diff
@@ -120,13 +120,15 @@ internals.Request.prototype._read = function (size) {
 
     setImmediate(() => {
 
+        /* $lab:coverage:off$ */
         if (this._shot.isDone) {
             if (this._shot.simulate.end !== false) {        // 'end' defaults to true
```
Comment (mtharrison) [inline lib/request.js]: Can you wrap only this if statement please? We should have coverage on the outer if

**Comment by mcollina** [comment]
@mtharrison updated.

**Comment by hueniverse** [comment]
Why is this not covered in node 10?

**Comment by mtharrison** [comment]
I guess `_read()` isn't called twice whereas it was node < 10.

**Comment by mcollina** [comment]
Yes. Before, _read() was called for every asynchronous push(). Now it’s called only once.

(I think we can all agree that those lines are not very intuitive and this is a good change. It also fixes a bad bug with object streams).

**Comment by mcollina** [comment]
When do you plan to make a shot release with this fix? in CITGM we pull from npm.

**Comment by mtharrison** [comment]
I can do one right now, assumed it was using master.

**Comment by mtharrison** [comment]
v4.0.5 released

**Pull request merged**